src/modules/dgt.power.push/Logic/WebresourcesProcessor.cs

```diff
@@ -149,62 +191,60 @@ public void UpsertResources(bool publish)
         }
     }
 
-    private void AddResourceToSolution(Guid resourceId, string resourceName)
+    private void AddResourceToSolution(Guid resourceId, string resourceName, string solutionName)
     {
-        // Web Resource = 61
         var addReq = new AddSolutionComponentRequest
         {
             AddRequiredComponents = false,
-            ComponentType = 61,
+            ComponentType = SolutionComponent.Options.ComponentType.WebResource,
             ComponentId = resourceId,
-            SolutionUniqueName = _solutionName
+            SolutionUniqueName = solutionName
         };
 
         try
         {
-            AnsiConsole.MarkupLine(CultureInfo.InvariantCulture, "Add WebResource [green]{0}[/] to Solution [bold]{1}[/]", resourceName, _solutionName);
+            AnsiConsole.MarkupLine(CultureInfo.InvariantCulture, "Adding webresource [green]{0}[/] to solution [bold]{1}[/]", resourceName, solutionName);
             _connection.Execute(addReq);
+
+            AnsiConsole.MarkupLine(CultureInfo.InvariantCulture, $" {Emoji.Known.RightArrow} [green]Added[/]");
         }
         catch (Exception ex)
         {
-            try
-            {
-                _connection.Delete(WebResource.EntityLogicalName, resourceId);
```

Comment:
Trying to delete the webresource just if adding to solution fails does not seem logical for me. If there is a use case for it, we might want to keep it